docs/docs-ref-autogen/excel/excel.worksheet.yml

```diff
@@ -452,11 +452,11 @@ items:
           await Excel.run(async (context) => {
               const sheet = context.workbook.worksheets.getActiveWorksheet();
 
-              // To create a Range with multiple child areas, pass a 
+              // To create a RangeAreas object with multiple child areas, pass a 
               // comma-delimited string of range addresses to 
-              // Worksheet.getRange.
-              const range = sheet.getRange("F3:F5, H3:H5");
-              range.format.fill.color = "pink";
+              // Worksheet.getRanges.

```

Comment:
Is this feature out now? @Rick-Kirkham - I thought areas we coming in 1.9

Reply:
Yes, it's 1.9, but 1.9 is in preview, and this snippet is in the Preview section of the Excel snippets in Script Lab. I think this is the way we have always handled snippets for Preview APIs.

Reply:
Correct. However, the snippet was mapped to an earlier version function in Worksheet. I've adjusted the mapping file in [this PR](https://github.com/OfficeDev/office-js-snippets/pull/194). The snippet will stay in Preview as is, it just won't now get copied to our reference docs.

Reply:
@AlexJerabek My bad, I didn't notice this was not the snippets repo.